textgrad/engine/__init__.py

```diff
@@ -1,11 +1,19 @@
 from .base import EngineLM, CachedEngine
-
+from transformers import AutoConfig
 __ENGINE_NAME_SHORTCUTS__ = {
     "opus": "claude-3-opus-20240229",
     "haiku": "claude-3-haiku-20240307",
     "sonnet": "claude-3-sonnet-20240229",
     "together-llama-3-70b": "together-meta-llama/Llama-3-70b-chat-hf",
 }
+def is_in_hf(engine_name):
```

Comment:
I think we may want to let people use HF models through other non-VLLM infra as well, in case they have preferences. This condition may be too general. Can we make these something like `vllm-{engine_name}` ?